internal/backup/volumesnapshotbackup_action.go

```diff
@@ -36,6 +36,14 @@ func (p *VolumeSnapshotBackupBackupItemAction) Execute(item runtime.Unstructured
 	}
 	p.Log.Infof("Converted Item to VSB: %v", vsb)
 
+	// check the VSB has the same backup name from label as the current backup
+	isVSBForCurrentBackup := util.VSBBelongsToBackup(backup.Name, &vsb, p.Log)
+
+	if !isVSBForCurrentBackup {
+		p.Log.Infof("unrelated volumesnapshotbackup found %s", vsb.Name)
```

Comment:
```suggestion
		p.Log.Infof("unrelated volumesnapshotbackup found %s, skipping", vsb.Name)
```

Reply:
"unrelated volumesnapshotbackup found %s, skipping datamover backup for this VSB"

Reply:
@shubham-pampattiwar that's right -- we're not skipping backing up the resource, but we're skipping datamover-related actions